tests/FINDOLOGIC/Export/Tests/CSVSerializationTest.php

```diff
@@ -325,4 +325,22 @@ public function testAddingRelativeUrlIsNotCausingAnException()
 
         $this->assertEquals($imageWithRelativePath, $image->getCsvFragment());
     }
+
+    public function testNewLineCharactersAreRemoved()
```

Comment:
* Perhaps you can generalize this test by using `illegalPropertyProvider` (which should be renamed then, because it no longer only refers to illegal characters in property keys, but problematic ones in general.
* Is it sufficient to only cover `Description`? The comment says it is, but that's only true if we can be sure that the implementation detail of all calling the same `sanitize()` method will always be true, which is not guaranteed in any solid way whatsoever.
* If covering only `Description`, the method name should reflect that.

Reply:
Added a data provide vor some of the elements which do not have specific checks for integer or float values. :+1: